src/Config/Config.ts

```diff
@@ -257,6 +266,7 @@ interface BridgeWidgetConfigYAML {
     branding?: {
         widgetTitle: string,
     }
+    openIdOverrides?: Record<string, string>;
```

Comment:
I think the proper value type for most `Records` includes `undefined`.
TypeScript won't add this automatically, but obviously most Objects will return you `undefined` on many keys.

```suggestion
    openIdOverrides?: Record<string, string|undefined>;
```